src/modules/aws.rs

```diff
@@ -122,6 +128,7 @@ fn get_credentials_duration(
     aws_creds: &AwsCredsFile,
 ) -> Option<i64> {
     let expiration_env_vars = [
+        "AWS_SSO_SESSION_EXPIRATION",
         "AWS_CREDENTIAL_EXPIRATION",
         "AWS_SESSION_EXPIRATION",
         "AWSUME_EXPIRATION",
```

Comment:
I'm not sure if `AWS_SSO_SESSION_EXPIRATION` should be first here, because `AWSUME_EXPIRATION` is last. On the other hand, other options prefer the non-aws vars.